### PR TITLE
[Arm64/Unix] Add JIT_Stelem_Ref helper

### PR DESCRIPTION
**Pull request by sdmaclea**

### CODE REVIEW

**Comment by sdmaclea** [comment]
@janvorli @jkotas PTAL

@dotnet/arm64-contrib FYI

File: src/vm/arm64/cgencpu.h [hunk 1]
```diff
@@ -79,6 +79,18 @@ typedef INT64 StackElemType;
 // !! This expression assumes STACK_ELEM_SIZE is a power of 2.
 #define StackElemSize(parmSize) (((parmSize) + STACK_ELEM_SIZE - 1) & ~((ULONG)(STACK_ELEM_SIZE - 1)))
 
+
+#ifdef FEATURE_PAL // TODO-ARM64-WINDOWS Add JIT_Stelem_Ref support
```
Comment (jkotas) [inline src/vm/arm64/cgencpu.h]: Could you please open an issue on this?

Comment (sdmaclea) [reply src/vm/arm64/cgencpu.h]: Done.  #13590.  Pankaj will probably look at this.

**Comment by sdmaclea** [comment]
@dotnet-bot test Tizen armel Cross Debug Build

**Comment by sdmaclea** [comment]
@briansull @jkotas Tests are clean this can be merged

**Comment by jkotas** [comment]
Thanks

**Pull request merged**